index.html

```diff
@@ -50,26 +50,27 @@
       </form>
```

Comment:
For simple UI components like search field, there is no need to include the `<form>` tag. See [this](https://stackoverflow.com/questions/3294572/is-input-well-formed-without-a-form)

Reply:
Thanks sir, the suggestions are very helpful.